src/three-components/SmoothControls.ts

```diff
@@ -356,13 +356,10 @@ export class SmoothControls extends EventDispatcher {
     // Re-evaluates clamping based on potentially new values for min/max
     // polar, azimuth and radius:
     this.setOrbit();
+    this.setFieldOfView(Math.exp(this[$goalLogFov]));
```

Comment:
Can you offer more detail for this change?

Reply:
Sure, I had to update it since goalSpherical didn't include field of view. Then I realized that this was just doing exactly what our jumpToGoal method was intended for; I must have just forgotten to update it when I added that method. It already contains the isStationary check, as well as all the field of view behavior.